### DISCUSSION TITLE
GH Projects

### DISCUSSION THREAD

**Discussion by Confectrician** [openhab/openhab-docs#2052] [Team Posts]
Hi,

I have already asked this some months ago in the community:
https://community.openhab.org/t/have-we-already-discussed-about-working-with-github-projects/33467?u=confectrician
There was no real discussion and the topic became inactive very fast.

Of course it is a bit unfortunate that only maintainters can create and edit projects.
But maybe this would be a good way for us 3 to document things like the linter refactoring over several repositories.

Wdyt?
Should we give it a try with that linter stuff?

**Comment by ThomDietrich** [comment]
Hey @Confectrician,
I'e tried projects in the past but it didn't stick. In general I think it's a cool thing but people would need to use it and that is rarely the case. Between the three of us I wouldn't mind trying it out for one task, here the linting and rewriting of all the binding READMEs. Feel free to dive in and get started!

**Comment by bgilmer77** [comment]
I have no idea how the projects work and for now I am focusing on the linter work, but I am very happy if you go ahead with this, and if there is something I need to do as I am working through the bindings, just let me know.

**Comment by Confectrician** [comment]
Ok. I will then dive into it.

> but people would need to use it and that is rarely the case.

Exactly that's the point, but if only 3 people have to use it for testing this should be better than 10 or more people who need to use it.

@bgilmer77 
My idea is to create 'linting-refactoring' project where all of us 3 can link issues or pull requests.
When we all update the project regularly everyone knows the global state of the linting refactoring and even the opened pull requests.

> if there is something I need to do
Only updating the project, when it exists and you do something.

So the main benefit i would like to have is a nice overview for all of us.
Those discussions here are a good start, but they're not made for such an overview.


(Of course i am perfectly fine with stopping projects, if they don't bring the benefit i/we would like to see.)

**Comment by bgilmer77** [comment]
I am not focused on this right now, but @Confectrician if you tell me what to do, i will do it.

**Comment by Confectrician** [comment]
I will do so, when the porject has been set up by me.

**Comment by bgilmer77** [comment]
Great!
